src/resources/views/crud/inc/datatables_logic.blade.php

```diff
@@ -7,12 +14,28 @@
   <script type="text/javascript" src="{{ asset('packages/datatables.net-fixedheader-bs4/js/fixedHeader.bootstrap4.min.js') }}"></script>
 
   <script>
+
+    // in this page we allways pass the alerts to localStorage because we can be redirected with
+    // persistent table, and this way we guarantee non-duplicate alerts.
+    $oldAlerts = JSON.parse(localStorage.getItem('backpack_alerts')) ?? {};
+    $newAlerts = @json($backpack_alerts);
+
+    Object.entries($newAlerts).forEach(([type, msg]) => {
+        if(typeof $oldAlerts[type] === undefined) {
+            $oldAlerts[type].push(msg);
+        }else{
+            $oldAlerts[type] = msg;
+        }
+    });
```

Comment:
Again, shouldn't we push to the array, when `!== undefined`?
I'm asking because I wasn't able to reproduce this 😅

Reply:
Totally, just fixed!